File: code/97113.pas
program Bai113;
var xK, yK, xM, yM, r, a: real;
function KC(x1, x2, y1, y2: real): real;
  var d : real;
  begin
  d := sqrt(sqr(x2 - x1) + sqr(y2 - y1));
  end;
  begin
  readln(xK, yK, r);
  readln(xM, yM);
  a := KC(xK, xM, yK, yM);
     if r = a then writeln('NAM TREN') 
   else  if a < r then writeln('NAM TRONG') else writeln('NAM NGOAI');
   end.

File: code/97113.cpp
#include <iostream>
#include <cmath>
using namespace std;

int sqr(int n){
  return n*n;
}

int main() 
{
    int xk,yk,xm,ym,r;
    cin>>xk>>yk>>r;
    cin>>xm>>ym;
    float km;
    km=sqrt(sqr(xm-xk)+sqr(ym-yk));
    if (km==r)
        cout<<"TREN";
    if (km>r)
        cout<<"NGOAI";
    if (km<r)
        cout<<"TRONG";
    return 0;
}